packages/@okta/vuepress-site/docs/guides/set-up-self-service-registration/overview/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Overview
+---
+
+A new user's first impression starts with the registration process, making it a critical aspect to get right. Because you're asking a user to give you information for the first time, you need to personalize and streamline the experience.
+
+Okta's self-service registration lets you configure a custom app or the Okta Homepage for use when users self-register. After you configure and enable your self-service registration policy, a sign-up link appears in the Okta Sign-In Widget. Users who click this link are directed to a Create Account registration form that is based on your registration policy.
+
+**Self-service registration supports these registration workflows:**
+
+* **Make email verification mandatory.** After registering their information, users are immediately sent an email to verify their email address. Users must click the link in the email to complete the registration process. Users are then redirected to your app or to your org's Okta Homepage.
+
+> **Note:** The lifetime of the emailed link is dictated by the **Activation emails are valid for** setting on the **Security** > **General** page of the Admin Console.
```

Comment:
The Developer console doesn't have a "Security" page. The user needs to switch to the "classic" view

Reply:
> The Developer console doesn't have a "Security" page. The user needs to switch to the "classic" view

The Admin Console is what we call the "Classic View". The Dev Console is what we call the Developer console. I can put "Classic UI" in parens if you think it'll help with the confusion.